Utility/Connect-AzureVM.ps1

```diff
@@ -27,103 +27,198 @@
 .RELEASENOTES
 
 
-#> 
-
-#Requires -Module Azure
-
-
-
-<# 
-
-.DESCRIPTION 
-    This runbook sets up a connection to an Azure virtual machine. It requires the Azure virtual machine to
-    have the Windows Remote Management service enabled, which is the default. It sets up a connection to the Azure
-	subscription and then imports the certificate used for the Azure VM so remote PowerShell calls can be made to it.  
-
-#> 
-Param()
-
+#>
 
 <#
 .SYNOPSIS 
-    Sets up the connection to an Azure VM
+    Sets up the connection to an Azure ARM VM
 
 .DESCRIPTION
-    This runbook sets up a connection to an Azure virtual machine. It requires the Azure virtual machine to
-    have the Windows Remote Management service enabled, which is the default. It sets up a connection to the Azure
-	subscription and then imports the certificate used for the Azure VM so remote PowerShell calls can be made to it.  
+    This runbook sets up a connection to an Azure ARM virtual machine. It requires the Azure virtual machine to
+    have the Windows Remote Management service enabled. It enables WinRM and configures it on your VM after which it sets up a connection to the Azure
```

Comment:
Why do we say it requires te WinRM service to be enabled, andthen right after say it enables WinRM? Which one is it?